week-5/DaInglichTim/queries/README.md

```diff
@@ -0,0 +1,77 @@
+# Solutionary
+
+## Show the total amount of sales in January 2009
+
+```sql
+SELECT COUNT(id_transaction)
+FROM transactions;
+```
+
+## Build a report of number of sales and total amount by each B2B customer
+
+```sql
+SELECT t.sold_by, COUNT(*), SUM(p.price)
+FROM transactions as t
+  INNER JOIN products as p on t.id_product = p.id_product
+GROUP BY t.sold_by;
+```
+
+## What is your best product?
+
+```sql
+SELECT p.product, COUNT(*) as counter
+FROM transactions as t
+  INNER JOIN products as p on t.id_product = p.id_product
+GROUP BY p.product
+ORDER BY counter DESC LIMIT 1;
+```
```

Comment:
First of all I think the question is ambiguous. You are considering that the best product is the most requested one, but some people can say that the best product is the one that generates more money.

Reply:
You rigth!!, but we chose the best product as the most requested one. :smile: